packages/ez-core/src/utils/map.ts

```diff
@@ -62,17 +63,33 @@ export const getGeoFeatureDataDict = (
     const datum = data.find(
       datum => datum[geoDomainKey] === properties[geoDomainKey]
     );
+
     return {
       feature,
       datum,
     };
   }, {} as GeoFeatureDataDict);
 };
 
+export let sqrtScale = d3Scale
+  .scaleSqrt()
+  .domain([0, 100])
+  .range([0, 30]);
+
+export const scaler = (rangeMin: number, rangeMax: number, value: number) => {
+  const sqrtScale = d3Scale
+    .scaleSqrt()
+    .domain([0, 100])
+    .range([rangeMin, rangeMax]);
+
+  return sqrtScale(value);
+};
+
 export const generateGeoFeaturePath = (
   feature: GeoFeature,
   projectionType: GeoProjectionType,
-  { center, scale, offset }: GeoProjectionCenter
+  { center, scale, offset }: GeoProjectionCenter,
```

Comment:
We should leave the generateGeoFeaturePath() function like it was. It just generates a path. We should add the centroid as a new attribute to the GeoFeatureDatum type and calculate + add it in the scaleGeoFeatureData() function in core/src/utils/scale.ts just like the other shapes.